app/controllers/categories_controller.rb

```diff
@@ -341,6 +341,12 @@ def search
       else
         true
       end
+    include_ancestors =
+      if params[:include_ancestors].present?
+        ActiveModel::Type::Boolean.new.cast(params[:include_ancestors])
+      else
+        true
```

Comment:
Would be good to add a controller test for this, i.e. that confirms that endpoint defaults to including ancestors.